paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md

```diff
@@ -0,0 +1,88 @@
+# INT8 MKLDNN quantization 
+
+This document describes how to use Paddle inference to convert the FP32 model to INT8 model on ResNet-50 and MobileNet-V1. We provide the instructions on enabling INT8 MKLDNN quantization in Paddle inference and show the ResNet-50 and MobileNet-V1 results in accuracy and performance.
+
+## 0. Prerequisite
+You need to build install at least PaddlePaddle-1.4.The test dataset and model will downloaded automatically in the build process.
```

Comment:
please put the build and install command. You can refer to [1. Install PaddlePaddle]
https://github.com/PaddlePaddle/models/blob/6cf68df0ca9d507a13b76c3531103b1d99cdf3d3/fluid/PaddleCV/image_classification/README_ngraph.md#1-install-paddlepaddle

Reply:
Done